issues-workflow.md

```diff
@@ -27,6 +27,7 @@ Here are the custom labels introduced by the Kyma team:
 * `priority/critical` indicates the top-priority of a given issue.
 * `area/{CAPABILITY_NAME}` indicates which capabilities are related to a given issue. You can assign more than one `area` label an issue.
 * `security/{SCORE}` indicates the security issue based on the [CVSSv3](https://www.first.org/cvss/calculator/3.0) security rating.
+* `{SIG-WG-LABEL}` indicates which SIG or WG identified the issue and is responsible for further follow-up of the issue.
```

Comment:
shouldnt be `sig/{sig-name}`? e.g `sig/core`?

Reply:
@mjasinski5 was thinking about it. We have 2 concepts SIGs and WGs. 
So we get to 

*sig/core
wg/xyz*

or we simply have a standard for SIG/WG short names (as we implicitly have) and we only use the name such as:

sig-core
wg-xyz

Personally, both are fine, the second is closer to naming as we use it in slack channels, DLs etc.

Reply:
naming first of all must be consistent with the labels naming we already have, so it should be:
- sig/core
- wg/consolidation

Reply:
So my suggestion is to have 2 separate points:
* sig/{sig-name} - e.g `sig/core`
* wg/{wg-name} - e.g `wg/consolidation`

as `{SIG-WG-LABEL}` doesnt tell the user how it should be named